packages/mcp/src/server.ts

```diff
@@ -211,24 +212,40 @@ export class MCPServer extends MCPServerBase {
    * @param req Incoming HTTP request
    * @param res HTTP response (must support .write/.end)
    */
-  public async startSSE({ url, ssePath, messagePath, req, res }: MCPServerSSEOptions): Promise<void> {
+  public async startSSE({ url, ssePath, messagePath, stream, context }: MCPServerSSEOptions) {
```

Comment:
Maybe we should make a new `startHonoSSE`? Since the old req/res might still be used by someone that's using express